src/main/java/org/prebid/server/analytics/reporter/greenbids/model/CommonMessage.java

```diff
@@ -0,0 +1,42 @@
+package org.prebid.server.analytics.reporter.greenbids.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Value;
+
+import java.util.List;
+
+@Builder(toBuilder = true)
+@Value
+public class CommonMessage {
+
+    @JsonProperty("version")
+    private final String version;
```

Comment:
related to https://github.com/prebid/prebid-server-java/pull/3096#discussion_r1628094156

Reply:
it's not necessary to annotate if the field name is a single lowercased word otherwise it's better to check whether our jackson mapper serialize in a way you require

Reply:
see continue of discussion https://github.com/prebid/prebid-server-java/pull/3096#discussion_r1629193104